activesupport/lib/active_support/message_pack.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+gem "msgpack", ">= 1.7.0"
```

Comment:
We may want to "decorate" that error like we do for cache backends: https://github.com/rails/rails/blob/c396d975350dfd042ba5d14ef83013184ff2f5e7/activesupport/lib/active_support/cache/mem_cache_store.rb#L3-L8

Reply:
I've added a hint now.  It's worth noting that `Gem::LoadError` (via the `gem` call) also includes a hint in its error message:

> ```ActiveSupport::MessagePack requires the msgpack gem, version 1.7.0 or later. Please add it to your Gemfile: `gem "msgpack", ">= 1.7.0"`​```
> ```.../bundler-2.4.12/lib/bundler/rubygems_integration.rb:280:in `block (2 levels) in replace_gem': msgpack is not part of the bundle. Add it to your Gemfile. (Gem::LoadError)```

We appear to use a mix of styles across the code base: [`require` with `rescue`](https://github.com/rails/rails/blob/c396d975350dfd042ba5d14ef83013184ff2f5e7/activesupport/lib/active_support/cache/mem_cache_store.rb#L3-L8), [`gem` with `rescue`](https://github.com/rails/rails/blob/bd8aeead92c11dbd82ddb9f114ea63b0daf160b4/activesupport/lib/active_support/cache/redis_cache_store.rb#L3-L10), and [`gem` without `rescue`](https://github.com/rails/rails/blob/bd8aeead92c11dbd82ddb9f114ea63b0daf160b4/activerecord/lib/active_record/connection_adapters/mysql2_adapter.rb#L6-L7).  It might be worth standardizing a single approach, but that can be tackled in another PR.